docs/user-interface/images/images.md

```diff
@@ -81,3 +81,14 @@ A background color for an image can also be specified:
 
 <!-- Valid color values are actually derived from the SKColor struct, rather than Microsoft.Maui.Graphics.Colors. -->
 Color values can be specified in hexadecimal, or as a .NET MAUI color. For example, `Color="Red"` is valid.
+
+## Control image quality by specifying FilterQuality
+
+The underlying [SKFilterQuality](https://github.com/mono/SkiaSharp/blob/f9bf4512767c9de22b6683e573b145ce5f0ce0a9/binding/SkiaSharp/SKPaint.cs#L17) used to compress the images is set to `SKFilterQuality.High` by default.
+
+With https://github.com/dotnet/maui/pull/25686 a new configuration property `FilterQuality` was introduced that can be set to any member of `SKFilterQuality`:
```

Comment:
do we want to use the wording "property" here, when the end user will need to specify an "attribute" on the MauiImage Item? 

I know the property was added in the class but that is an internal class which the user will not be able to use. 
And we don't want to confuse it with an MSBuild property.

Reply:
Good point. Changed.